File: swt/MFD/SRC/STD.R/CLEAR.R
# clear -- clear a users terminal screen

   integer vtterm, vt$clr
   integer i, term_type (MAXTERMTYPE)

   if (vtterm (term_type) == ERR || vt$clr (i) == ERR)
      for (i = 1; i <= 25; i += 1)
         call putch (NEWLINE, STDOUT)

   stop
   end

File: swt/src/std.r/clear.f
      INTEGER VTTERM,VT$CLR
      INTEGER I,TERMT0(7)
      IF((VTTERM(TERMT0).EQ.-3))GOTO 10001
      IF((VT$CLR(I).EQ.-3))GOTO 10001
      GOTO 10000
10001   I=1
        GOTO 10004
10002   I=I+(1)
10004   IF((I.GT.25))GOTO 10003
          CALL PUTCH(138,-11)
        GOTO 10002
10003 CONTINUE
10000 CALL SWT
      END
C ---- Long Name Map ----
C termtype                       termt0
